dev/src/write-batch.ts

```diff
@@ -194,12 +193,13 @@ export class WriteBatch {
 
     const op = () => {
       const document = DocumentSnapshot.fromObject(documentRef, firestoreData);
-      const write =
-        !document.isEmpty || transform.isEmpty ? document.toProto() : null;
```

Comment:
Can you also check if we still use `document.isEmpty`?

Reply:
removed.